### PR TITLE
Make eslint valid JSON

### PR DESCRIPTION
**Pull request by erikphansen**
Using single-quotes in JSON isn't valid. And my editor *loooooves* to tell me about it. This PR simply runs the `.eslintrc` through Prettier's default JSON formatter to convert the quotes to double-quotes.

The bigger reason for this PR is so that when I make the PR to implement Prettier's eslint rules those legit code changes don't get mixed in with this reformatting.

eslint-invalid-json

### CODE REVIEW

**Review by ncksllvn — APPROVED** [review]
LGTM!

**Review by jbalboni — APPROVED** [review]

**Review by cehsu — APPROVED** [review]
LGTM. I had a few questions that could be put into new issues if they are of interest.

File: .eslintrc [hunk 1]
```diff
@@ -1,122 +1,141 @@
...
+        "src/applications/static-pages/**/*.js",
+        "src/applications/static-pages/**/*.jsx"
+      ],
+      "rules": {
+        "no-restricted-imports": ["error", "lodash/fp"]
+      }
+    },
+    {
+      "files": [
+        "**/*.spec.jsx",
```
[the remark this answers is not in the document]
Comment (erikphansen) [reply .eslintrc]: that was Prettier, not me :)

File: .eslintrc [hunk 2]
```diff
@@ -1,122 +1,141 @@
...
-    'rules': {
-      'no-restricted-imports': ['error', 'lodash/fp']
+  "overrides": [
+    {
+      "files": [
+        "src/platform/**/*.js",
+        "src/platform/**/*.jsx",
+        "src/applications/site-wide/**/*.js",
+        "src/applications/site-wide/**/*.jsx",
+        "src/applications/static-pages/**/*.js",
```
[the remark this answers is not in the document]
Comment (erikphansen) [reply .eslintrc]: Mmmmmmmaybe? I would be concerned of unintended side effects. I kinda prefer to be verbose and clear about the intention here.

File: .eslintrc [hunk 3]
```diff
@@ -1,122 +1,141 @@
...
+    ], // TODO(awong): Too hard to turn on.
+    "react/jsx-sort-props": [
+      0,
+      { "callbacksLast": true, "shorthandFirst": true }
+    ], // TODO(awong): Too hard to turn on.
+    "react/jsx-no-duplicate-props": 2,
+    "react/no-danger": 2,
+    "react/no-deprecated": 2,
+    "react/no-direct-mutation-state": 2,
+    "react/no-string-refs": 0, // TODO(awong): Enable.
```
[the remark this answers is not in the document]
Comment (erikphansen) [reply .eslintrc]: I don't know the context here, but making any changes to the rules is out of scope for this PR's intent, which is just reformatting. But I'm also reworking the actual rules in my big PR that will follow this and will check out all the rules marked with `TODO` and see if they could be turned on.

[the remark this answers is not in the document]
Comment (cehsu) [reply .eslintrc]: Thanks for the context Erik, it's good to hear that someone is doing a deep dive into this.

**Review by rianfowler — APPROVED** [review]

**Pull request merged**